.github/workflows/ci.yml

```diff
@@ -31,4 +42,15 @@ jobs:
     - name: Install clippy
       run: rustup component add clippy
     - name: Run clippy
-      run: cargo clippy --all-features
+      run: cargo clippy
+  clippy-borsh:
```

Comment:
Let’s run clippy only for all features enabled to save CI resources

Reply:
int.checked_mul(percent as u128).unwrap_or_else(|| env::panic_str("royalty overflow")) / 100u128